### DISCUSSION TITLE
FreshRSS

### DISCUSSION THREAD

**Discussion by isabelroses** [catppuccin/catppuccin#1938] [Port Requests]
### Will you make the port?

- [x] Yes, I'm willing to make the port

### How can the requested port be themed?

Link to GitHub: https://github.com/FreshRSS/FreshRSS
From here you can learn the theming for the application.

### Has the requested port been themed already?

Link to my port: https://github.com/isabelroses/freshrss-catppuccin 
I am willing to make changes

**Comment by sgoudham** [comment]
Hiya :wave:, just have some preliminary comments before moving this to be reviewed. 

Can you please ensure that you're adhering to our newly merged [submission guidelines](https://github.com/catppuccin/catppuccin/blob/main/docs/port-creation.md#submission)? Specifically our [styling guidelines](https://github.com/catppuccin/catppuccin/blob/main/docs/port-creation.md#styling) as I can notice some things being off!

Thank you for your work so far! :heart: 
Feel free to comment if something is unclear and I'll try my best to explain!

**Reply by isabelroses** [reply]
Haii.

Thanks for the advice. 
I have made the appropriate changes and read over the guidelines.
I will be more then happy to continue maintenance as well and create other form of the port.

Thanks for your help and looking over this port.

**Reply by sgoudham** [reply]
Thanks! Transferred it over to #1962 for port review :heart:
